DQM/L1TMonitor/src/L1TdeStage2EMTF.cc

```diff
@@ -1,11 +1,15 @@
 #include <string>
-
+#include <vector>
 #include "DQM/L1TMonitor/interface/L1TdeStage2EMTF.h"
 
 
 L1TdeStage2EMTF::L1TdeStage2EMTF(const edm::ParameterSet& ps)
     : dataToken(consumes<l1t::RegionalMuonCandBxCollection>(ps.getParameter<edm::InputTag>("dataSource"))),
       emulToken(consumes<l1t::RegionalMuonCandBxCollection>(ps.getParameter<edm::InputTag>("emulSource"))),
+      datahitToken(consumes<l1t::EMTFHitCollection>(ps.getParameter<edm::InputTag>("dataSource"))),
+      emulhitToken(consumes<l1t::EMTFHitCollection>(ps.getParameter<edm::InputTag>("emulSource"))),
+      datatrackToken(consumes<l1t::EMTFTrackCollection>(ps.getParameter<edm::InputTag>("dataSource"))),
+      emultrackToken(consumes<l1t::EMTFTrackCollection>(ps.getParameter<edm::InputTag>("emulSource"))),
       monitorDir(ps.getUntrackedParameter<std::string>("monitorDir", "")),
```

Comment:
To fix the ProductNotFound error every token should have its own InputTag in order to choose different instance names. They should not use all `emtfSource` and `dataSource`.